src/applications/personalization/dashboard-2/utils/mocks/MOCK_VA_APPOINTMENTS.js

```diff
@@ -0,0 +1,1000 @@
+const oneDayInMS = 24 * 60 * 60 * 1000;
+
+export default [
+  {
+    id: '20abc6741c00ac67b6cbf6b972d084c1',
+    type: 'va_appointments',
+    attributes: {
+      startDate: Date.now() + oneDayInMS * 7,
```

Comment:
This appointments mock data is a copy of the mock data in vaos, with the exception of the `startDate` property which I made dynamic for testing. Since we only want to render an appointment update if it's scheduled within 30 days, we should not hard code this date.